src/renderer/components/dock/dock-tab.store.ts

```diff
@@ -1,25 +1,30 @@
 import { autorun, observable, reaction } from "mobx";
-import { autobind, createStorage } from "../../utils";
+import { autobind, createStorage, StorageHelper } from "../../utils";
 import { dockStore, TabId } from "./dock.store";
 
 interface Options<T = any> {
-  storageName?: string; // name to sync data with localStorage
-  storageSerializer?: (data: T) => Partial<T>; // allow to customize data before saving to localStorage
+  storageName?: string; // persistent key
```

Comment:
The key isn't persistent but the data behind the key

```suggestion
  storageName?: string; // persistence key
```